mytimer/functions.py

```diff
@@ -313,6 +313,14 @@ def countdown_timer(
         time.sleep(max(0, 1 - (end - start)))
 
 
+def show_faces_list():
```

Comment:
You can move this function next to the `show_programs_list` for more readability and cohesion.